packages/building-rollup/modern-config.js

```diff
@@ -25,6 +27,7 @@ module.exports = function createBasicConfig(_options) {
     outputDir: 'dist',
     extensions: DEFAULT_EXTENSIONS,
     indexHTMLPlugin: {},
+    dedupeModules: [],
```

Comment:
we should stick to only flags to disable plugins entirely, and document how people can add their own configuration.

in this case it's easy to add your own copy of node-resolve I think